integrations/operator/README.md

```diff
@@ -24,6 +24,7 @@ The operator supports reconciling the following Kubernetes CRs:
 - TeleportOpenSSHEICEServerV2
 - TeleportOpenSSHServerV2
 - TeleportTrustedClusterV2
+- TeleportBot
```

Comment:
Since [RFD 160](https://github.com/gravitational/teleport/pull/35954) all CRs should be suffixed with the Teleport resource version.

`TeleportBot` should be `TeleportBotV1`. This should be reflected everywhere: resource kind, CR types, user-fcing docs, go file name, ...